src/App.jsx

```diff
@@ -1,40 +1,40 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
-export const tabs = [
+const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
+export const App = () => {
+  const defaultTab = tabs[0];
+  const [selectedTabId, setTabId] = useState('');
```

Comment:
1) no needs to create an extra variable defaultTab
2) set first tab as a default to a state
3) rename a little

```suggestion
  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```